resources/production/config-example.toml

```diff
@@ -5,7 +5,7 @@
 
 # If set, use this hash as a trust anchor when joining an existing network.
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
-
+archival_sync = false
```

Comment:
```suggestion
archival_sync = true
```

Reply:
With the current implementation, this will cause nodes to take too long to join.

After implementing your proposal I have outlined in https://github.com/casper-network/casper-node/issues/2083 we can set these flags to `true`.

Reply:
Same here, archival syncing should be enabled by default.

Reply:
Done in 209c9e95933dd452f349c0b9c1380487ab221d16